.github/workflows/nightly-test-integrations-1.15.x.yml

```diff
@@ -204,7 +207,12 @@ jobs:
         consul-version: ["1.14", "1.15"]
     env:
       CONSUL_LATEST_VERSION: ${{ matrix.consul-version }}
-      ENVOY_VERSION: "1.24.6"
+      # ENVOY_VERSION should be the latest version supported by all
+      # Consul versions in the matrix.consul-version, since we are testing upgrade from
+      # an older Consul version. In practice, this should be the highest Envoy version
+      # supported by the lowest non-LTS Consul version in the matrix (LTS versions receive
+      # additional Envoy version support).
+      ENVOY_VERSION: "1.24.12"
```

Comment:
I skipped on updating this version since 1.24 is frozen, 1.15 is LTS (so supports much later versions), and 1.14 won't get a backport of `ENVOY_VERSIONS`